src/hooks/useReadNextExercise.ts

```diff
@@ -3,7 +3,6 @@ import { Discipline } from '../constants/endpoints'
 import { getNextExercise } from '../services/helpers'
 import useLoadAsync, { Return } from './useLoadAsync'
 
-const useReadNextExercise = (profession: Discipline | null): Return<NextExercise | null> =>
-  useLoadAsync(getNextExercise, profession)
+const useReadNextExercise = (profession: Discipline): Return<NextExercise> => useLoadAsync(getNextExercise, profession)
```

Comment:
Isn't it possible that there is no next exercise?

Reply:
there is already a design ticket, so for the moment i keep it as it is